jupyter-notebook/Dockerfile

```diff
@@ -8,6 +8,7 @@ RUN apt update && \
     apt install --yes --no-install-recommends \
     git \
     git-lfs \
+    nodejs \
```

Comment:
Why do we need `nodejs`?

Reply:
Jupyter was complaining about compiling js code, but it works fine without, indeed.